src/App.js

```diff
@@ -1,26 +1,57 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React, { Component } from 'react'
+import Title from './Title'
+import jsondata from './content.json'
+import Foot from './foot'
+import Content from './content'
+import'./app.css'
+
+
+
+class App extends Component {
+
```

Comment:
Try to use hook instead of class component here?
You can get some ideas from the link below:
https://reactjs.org/docs/hooks-state.html